test/e2e/run_test.go

```diff
@@ -1406,6 +1406,13 @@ VOLUME %s`, ALPINE, volPath, volPath)
 		Expect(session.OutputToString()).To(ContainSubstring("1001-123"))
 	})
 
+	It("podman run --mount type=devpts,target=/dev/pts with ptmxmode", func() {
+		session := podmanTest.Podman([]string{"run", "-t", "--mount", "type=devpts,target=/dev/pts,ptmxmode=0666", fedoraMinimal, "findmnt", "-noOPTIONS", "/dev/pts/0"})
```

Comment:
should it be "/dev/pts" instead of "/dev/pts/0"?

Reply:
> should it be "/dev/pts" instead of "/dev/pts/0"?

Let's find out. I pushed a new version.